tests/src/test/java/alluxio/shell/command/ChownCommandIntegrationTest.java

```diff
@@ -46,13 +183,44 @@ public void chown() throws IOException, AlluxioException {
   public void chownRecursive() throws IOException, AlluxioException {
     clearLoginUser();
     FileSystemTestUtils.createByteFile(mFileSystem, "/testDir/testFile", WriteType.MUST_CACHE, 10);
-    mFsShell.run("chown", "-R", "user1", "/testDir");
+    mFsShell.run("chown", "-R", TEST_USER_1.getUser(), "/testDir");
     String owner = mFileSystem.getStatus(new AlluxioURI("/testDir/testFile")).getOwner();
-    Assert.assertEquals("user1", owner);
+    Assert.assertEquals(TEST_USER_1.getUser(), owner);
     owner = mFileSystem.getStatus(new AlluxioURI("/testDir")).getOwner();
-    Assert.assertEquals("user1", owner);
-    mFsShell.run("chown", "-R", "user2", "/testDir");
+    Assert.assertEquals(TEST_USER_1.getUser(), owner);
+    mFsShell.run("chown", "-R", TEST_USER_2.getUser(), "/testDir");
     owner = mFileSystem.getStatus(new AlluxioURI("/testDir/testFile")).getOwner();
-    Assert.assertEquals("user2", owner);
+    Assert.assertEquals(TEST_USER_2.getUser(), owner);
+  }
+
+  /**
+   * Run chown command to change owner and group of a path.
+   *
+   * @param command the command to run
+   * @param expectedReturnValue return value expected from running the command
+   * @param expectedCommandOutput command output expected from running the command
+   * @param expectedOwner expected owner of the path
+   * @param expectedGroup expected group of the path
+   */
+  private void runChownOwnerAndGroup(String path, int expectedReturnValue,
```

Comment:
This method uses a lot of parameters but has very little content, could we instead inline the calls to `verifyCommandReturnValueAndOutput` and `verifyCommandReturnValueAndOutput` in the individual test methods?